site/admin/theatres/create.php

```diff
@@ -0,0 +1,107 @@
+<?php
+include_once('init.php');
+PrintHeader('Create Theatre', 'Create Theatre');
+
+
+if(isset($_REQUEST['TheatreFormSubmitted'])){
+	//die("the form was submitted");
+	if(!@$_REQUEST['TheatreName']){
```

Comment:
This validation looks good.

For bonus points, you might try making a function that handles these common validation scenarios. Like, you could replace lines 8-10 with `ValidateText('TheatreName')` (and maybe have some other parameters for common things like minimum # of characters, etc.)